packages/p2p-v2/src/pages/my-ads/screens/MyAds/MyAdsTable/MyAdsTable.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { memo, useMemo } from 'react';
+import { p2p } from '@deriv/api';
+import { Loader } from '@deriv-com/ui';
+import { Table } from '../../../../../components';
+import { MyAdsTableRow } from '../MyAdsTableRow';
+import './MyAdsTable.scss';
+
+export type TMyAdsTableRowRendererProps = Partial<
+    NonNullable<ReturnType<typeof p2p.advertiserAdverts.useGet>['data']>[0]
+> & {
+    isBarred: boolean;
+    isListed: boolean;
+    onClickIcon: (id: string, action: string) => void;
+};
+
+const MyAdsTableRowRenderer = memo((values: TMyAdsTableRowRendererProps) => <MyAdsTableRow {...values} />);
+MyAdsTableRowRenderer.displayName = 'MyAdsTableRowRenderer';
+
+const headerRenderer = (header: string) => <span>{header}</span>;
+
+const MyAdsTable = () => {
+    const columns = useMemo(
+        () => [
+            {
+                header: 'Ad ID',
+            },
+            {
+                header: 'Limits',
+            },
+            {
+                header: 'Rate (1 BTC)',
+            },
+            {
+                header: 'Available amount',
+            },
+            {
+                header: 'Payment methods',
+            },
+            {
+                header: 'Status',
+            },
+        ],
+        []
+    );
```

Comment:
do we need to wrap this in useMemo? this never changes right? better to remove it out of the component so it never gets rerendered with the component